lte/gateway/python/integ_tests/defs.mk

```diff
@@ -54,6 +54,7 @@ s1aptests/test_sctp_abort_after_smc.py \
 s1aptests/test_sctp_shutdown_after_auth_req.py \
 s1aptests/test_sctp_shutdown_after_identity_req.py \
 s1aptests/test_sctp_shutdown_after_smc.py \
+s1aptests/test_sctp_shutdown_after_multi_ue_attach.py \
```

Comment:
Let's not add to sanity until we fix the bug...

Reply:
Removed the test case from sanity and Updated the PR